internal/kafka/internal/clusters/ocm_provider.go

```diff
@@ -125,6 +125,24 @@ func (o *OCMProvider) AddIdentityProvider(clusterSpec *types.ClusterSpec, identi
 	return nil, nil
 }
 
+func (o *OCMProvider) RemoveResources(clusterSpec *types.ClusterSpec, syncSetName string) error {
```

Comment:
Can we document this method so consumers of it understand the behavior? Specially the part where if a cluster does not exist it is not considered an error